### DISCUSSION TITLE
Bug! Projects in the new projects beta not assignable through issue URL

### DISCUSSION THREAD

**Discussion by mrmurphy** [community/community#18977] [Projects and Issues]
Hi there! I'm wanting to link a new issue to a project from the new Projects Beta by using a query parameter as described here: https://docs.github.com/en/enterprise-server@3.1/issues/tracking-your-work-with-issues/creating-an-issue

If I provide the ID of an old project: https://github.com/mrmurphy/RoadmapTool/issues/new?projects=mrmurphy/3
The project field on the issue gets properly assigned.

If I provide the ID of a project from the beta: https://github.com/mrmurphy/RoadmapTool/issues/new?projects=mrmurphy/1
The project field on the new issue is empty.

I assume that Beta Projects aren't yet supported through the URL?

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
